src/gitmoji.js

```diff
@@ -131,7 +131,7 @@ class GitmojiCli {
   }
 
   _commit (answers) {
-    const title = `${answers.gitmoji} ${answers.title}`
+    const title = `${answers.gitmoji} ${answers.scope ? ` (${answers.scope})-` : ' '}${answers.title}`
```

Comment:
You shouldn't be leaving the extra space if there's no scope provided:

```suggestion
    const title = `${answers.gitmoji} ${answers.scope ? ` (${answers.scope})-` : ''}${answers.title}`
```

Because if you take a look at the snapshot one extra space is generated:

https://github.com/carloscuesta/gitmoji-cli/pull/139/files#diff-c7c767aa8d5662bbf2b978bffe257b6aR45